concordion/src/test/resources/nl/knaw/huc/textrepo/index/TestFileIndexer.md

```diff
@@ -61,6 +61,41 @@ To check what version is the last version with changed contents, we can run the
 Full response:
 [ ](- "ext:embed=#result.body")
 
+# Other examples
 
+## Find files by type and latest modification time of contents
+
+Find all files by `{type}` that have versions with changed contents at or after `{dateTime}`:
+
+[ ](- "ext:embed=getEsQuerySearchByFileTypeAndContentsLastModified()")
+
+ - where `{dateTime}` is [ ](- "c:echo=getDateTime()")
+ - where `{type}` is [`text`](- "#type")
+
+[ ](- "#result=searchEsQuerySearchByFileTypeAndContentsLastModified(#type)")
+
+ - The response status should be: [200](- "?=#result.status");
+ - The response should contain [correct file](- "?=#result.found");
+
+Full response:
+[ ](- "ext:embed=#result.body")
+
+
+## Find documents by metadata
+
+Find all documents by document metadata `{key}` and `{value}`, including an aggregated count of found _documents_:
```

Comment:
Een usecase is mogelijk ook dat je alleen op document metadata 'key' wil zoeken (omdat er diverse mogelijke 'values' zijn, die je niet allemaal wilt opsommen, maar je wilt alleen documenten vinden waarvoor die 'key' gezet is)?

Reply:
Zal ik daarbij noemen dat we dit als 'tags' zouden kunnen beschouwen?

Reply:
(of staat dit los van onze 'tag-discussie'?)

Reply:
ja lijkt me prima!

Reply:
09ef66030dda9f63d44bc31580e0046d3a966bf7